ar-001/auth.json

```diff
@@ -0,0 +1,283 @@
+            "back_home_button": "العودة إلى الصفحة الرئيسية",
+            "recovery_codes_page_error_no_codes": "لا توجد رموز لإظهارها",
+            "recovery_codes_page_error_already_viewed": "يبدو أنك قد اطلعت على رموز الاسترداد مرة واحدة بالفعل، لذلك لا يمكننا إظهارها مرة أخرى.  إذا كنت بحاجة إلى مساعدة في تسجيل الدخول، فاتصل بمسؤول حسابك."
+        },
+        "authenticator_app_qr_code_page": {
+            "page_title": "مصادقة متعددة العوامل |  ${business_name}",
+            "heading": "تطبيق المصادقة",
+            "description": "امسح رمز QR ضوئيًا بمساعدة تطبيق المصادقة المفضل لديك.  أدخل رمز الإستخدام لمرة واحدة الذي ستتلقاه أدناه.",
+            "trouble_scanning_link": "هل تواجه مشكلة في المسح؟",
+            "code_input_label": "رمز الإستخدام لمرة واحدة ",
+            "code_input_error_required": "يرجى إدخال رمز تحقق صحيح",
+            "code_input_error_incorrect_code_length": "يجب أن يتكون رمز التأكيد من 6 أرقام",
+            "code_input_error_incorrect_code": "الرجاء إدخال رمز تأكيد صحيح",
+            "code_input_error_expired_code": "لقد انتهت صلاحية رمز التأكيد الخاص بك، يرجى طلب رمز جديد",
+            "continue_button": "المتابعة",
+            "choose_another_method_link": "إختر طريقة أخرى"
+        },
+        "authenticator_app_manual_page": {
+            "page_title": "تطبيق المصادقة |  ${business_name}",
+            "heading": "تطبيق المصادقة",
+            "description": "أدخل الرمز التالي في تطبيق المصادقة المفضل لديك.  ثم أدخل رمز الاستخدام لمرة واحدة الذي ستتلقاه أدناه.",
+            "code_input_label": "رمز الإستخدام لمرة واحدة ",
+            "code_input_error_required": "الرجاء إدخال رمز التحقق",
+            "code_input_error_incorrect_code_length": "يجب أن يتكون رمز التأكيد من 6 أرقام",
+            "code_input_error_incorrect_code": "الرجاء إدخال رمز تأكيد صحيح",
+            "code_input_error_expired_code": "لقد انتهت صلاحية رمز التأكيد الخاص بك، يرجى طلب رمز جديد",
+            "continue_button": "المتابعة",
+            "choose_another_method_link": "إختر طريقة أخرى"
+        },
+        "mfa_sms_page": {
+            "page_title": "أدخل رقم الهاتف |  ${business_name}",
+            "heading": "المصادقة عبر الرسائل القصيرة",
+            "description": "أضف رقم هاتف لتلقي الرموز عبر رسالة نصية",
+            "country_input_label": "الدولة او المنطقة",
+            "country_input_error_invalid": "مطلوب دولة صحيحة",
+            "phone_input_label": "رقم الهاتف",
+            "phone_input_error_required": "أدخل رقم الهاتف",
+            "phone_input_error_invalid_format": "أدخل الأرقام فقط بدون مسافات، باستثناء الصفر البادئ",
+            "phone_input_error_invalid_region": "الرقم غير صالح لهذه المنطقة",
+            "phone_input_help_text": "اترك الصفر البادئ",
+            "continue_button": "المتابعة",
+            "choose_another_method_link": "إختر طريقة أخرى"
+        },
+        "sms_verify_code_page": {
+            "page_title": "مصادقة متعددة العوامل |  ${business_name}",
```

Comment:
@onderay there looks like additional whitespace here on the right hand side of ` | `

Reply:
@DaveOrDead updated